content/verification/veteran-confirmation/release-notes/2024-05-14.md

```diff
@@ -0,0 +1,24 @@
+
+Special characters and spaces are now accepted in the Demographic fields.
+
+**_First, Middle, and Last Name fields_**
+
+Valid entries for the name fields include:
+* Upper and/or lower case letters
+* Spaces
+* Numbers
+* Special characters: forward slash /, back slash \, hyphen -, single quote '
+* More than one name in a field. For example: First name: Margaret Mary
+* Names in the field can be separated by a space or a valid special character (forward slash /, back slash \, hyphen -, single quote ').
+  
+**_zip, city_**
+
+Valid entries for zip and city include:
+* One to sixteen uppercase or lowercase letters, digits, spaces, or hyphens -.
```

Comment:
@camronmartinez is this a specific case where only one to sixteen upper and lower case letters etc. can be included? Or is it only for zip, city where 1 - 16 characters can be included?

Reply:
Zip can be 1 to 16 characters: accepts alphanumeric, spaces and hypens.
City doesn't have the the 1 to 16 character limit and has the additional symbols.